Makefile

```diff
@@ -94,6 +95,9 @@ clean-all:
 	sudo podman rmi -f "${USHIFT_IMAGE}" || true
 	sudo podman rmi -f "${BUILDER_IMAGE}" || true
 
+.PHONY: check
+check: _hadolint _shellcheck
```

Comment:
Although I think it's handy to have it, it seems this `check` target is never user.

Reply:
The check target is a "public" one (without _) to be used in dev flows.

Reply:
It's the one appearing in the make help message.